### ISSUE TITLE
`is_categorical_dtype` is deprecated

### ISSUE THREAD

**Issue by GegznaV** [fbdesignpro/sweetviz#162]
I use Sweetviz 2.2.1. I recently updated to Pandas 2.1.3 and started receiving the following message/warning:
```
is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
```
- See https://github.com/pandas-dev/pandas/pull/52527
If any additional details are needed, please let me know.

**Comment by fbdesignpro** [comment]
Noted, will look into implementing soon, thank you!

**Comment by fbdesignpro** [comment]
Fixed in 2.3.0, thanks again @GegznaV!
